pkg/notify/messages.go

```diff
@@ -44,6 +49,16 @@ func WrapTaskStateUpdate(tsu *TaskStateUpdate) *Message {
 	if tsu.PotentialResolvers != nil && len(tsu.PotentialResolvers) > 0 {
 		m.Fields["potential_resolvers"] = strings.Join(tsu.PotentialResolvers, " ")
 	}
+	if tsu.ResolutionPublicID != "" {
+		m.Fields["resolution_id"] = tsu.ResolutionPublicID
+	}
+
+	if tsu.Tags != nil {
+		tags, err := json.Marshal(tsu.Tags)
+		if err == nil {
+			m.Fields["tags"] = string(tags)
+		}
```

Comment:
Should probably log the error here even if we continue the norification process.